testing/kuttl/e2e/major-upgrade-missing-image/12--start-and-update-version.yaml

```diff
@@ -0,0 +1,17 @@
+---
+# Shutdown the cluster -- but without the annotation.
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: major-upgrade-empty-image
+spec:
+  shutdown: false
```

Comment:
:question: How does this field setting fit with the comment? `# Shutdown the cluster -- but without the annotation.`